astroquery/mast/cutouts.py

```diff
@@ -56,35 +68,67 @@ def _parse_cutout_size(size):
         either pixels or degrees.
     """
 
+    # This local variable will change to True if input cutout size exceeds recommended limits for TESS
+    limit_reached = False
+
+    # Checking 2d size inputs for the recommended cutout size
+    if isinstance(size, list) & (mission == 'TESS'):
+        if len(size) == 2:
+            if np.isscalar(size[0]):
+                size = [size[0] * u.pixel, size[1] * u.pixel]
+
+            with u.set_enabled_equivalencies(u.pixel_scale(21 * u.arcsec / u.pixel)):
+                limit_reached = (size * size[0].unit > 30 * u.pixel).any()
```

Comment:
For `set_enabled_equivalencies`, the entire list needs to be assigned a unit rather than individual elements, in order for a conversion to take place. Otherwise this line results in an error.

<img width="1063" alt="image" src="https://user-images.githubusercontent.com/32107699/230422647-4857e98d-a6ac-4a76-a29e-826162eae409.png">